backend/src/dal/base_entities/user.py

```diff
@@ -0,0 +1,4 @@
+from base import BaseEntity
+class User(BaseEntity):
```

Comment:
more fields

registration_date, ip_address, last_seen_browser, last_login_date, email, birthdate 
by having more data you have more leverage when data is an issue you just store the trash in somewhere you dont need..